src/StableHashTraits.jl

```diff
@@ -97,9 +97,14 @@ end
 orderproperties(::UseProperties{:ByOrder}, props) = props
 orderproperties(::UseProperties{:ByName}, props) = TupleTools.sort(props; by=string)
 function stable_hash_helper(x, hash, context, use::UseProperties)
-    return stable_hash_helper((k => getproperty(x, k)
-                               for k in orderproperties(use, propertynames(x))), hash,
-                              context, UseIterate())
+    vals = (k => getproperty(x, k) for k in orderproperties(use, propertynames(x)))
+    return stable_hash_helper(vals, hash, context, UseIterate())
+end
+
+struct UseTable end
+function stable_hash_helper(x, hash, context, ::UseTable)
+    vals = (k => v for (k, v) in zip(Tables.columnnames(x), Tables.columns(x)))
```

Comment:
I believe you will need this to correctly handle row-oriented tables:
```suggestion
    cols = Tables.columns(x)
    vals = (k => v for (k, v) in zip(Tables.columnnames(cols), cols))
```
Can test with
```
row_table = [(; a = 1, b=2), (;a = 2, b=3)]
```

Reply:
Ah yes. Thank you!